packages/morphic-eq-interpreters/src/model/primitives.ts

```diff
@@ -12,9 +13,9 @@ export const eqPrimitiveInterpreter: ModelAlgebraPrimitive1<EqURI> = {
   boolean: _ => new EqType(eqBoolean),
   string: _ => new EqType(eqString),
   number: _ => new EqType(eqNumber),
-  bigint: _ => new EqType({ equals: (x, y) => eqStrict.equals(x, y) }),
+  bigint: _ => new EqType({ equals: deepEqual }),
```

Comment:
No need for deepEqual overhead for primitives (let's get back to dumb strictEqual - applies to other changed instances).
Bigint tests coming in ( https://github.com/sledorze/morphic-ts/pull/66 ).